### ISSUE TITLE
How do you approach proposing new ideas/tools/tech to your team at work?

### ISSUE THREAD

**Issue by joeybuczek** [kentcdodds/ama#432]
Kent,

In your course “Testing React Applications” on frontendmasters.com, you were asked a question regarding setting up testing tools at work. Your response indicated that among the tooling transitions you made, your teammates were hesitant to switch based on the results of a prior attempt. Eventually it turned out to be a good thing, as you mentioned, but I’m sure there was a heap of skepticism beforehand.

My question is this: 
How do you approach presenting new ideas/tools/tech/etc. at work while trying to alleviate any possible fears/hesitation that might arise? 

While you may be sure of the benefits of the proposal, your team/management might not be as open or may be very set in their ways. Do you create a presentation? Supply documentation/material? Create a demonstration?

Thanks,
Joey

**Comment by kentcdodds** [comment]
Hi! I'm afraid I don't have time to give a super solid answer to all my open questions right now, so I just went through and answered all my AMA questions in the form of a video: https://www.youtube.com/watch?v=y23g8L8UtoQ&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u

A few things that folks can do to help make things better for others reading the questions and answers:

1. Link to when I start answering the question. Add `&t=xxmxxs` to the end of the URL like this: https://www.youtube.com/watch?v=y23g8L8UtoQ&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u&t=2m5s
2. Comment with links I referenced
3. Summarize what I talk about

Thanks!
